gapic/generator/options.py

```diff
@@ -28,7 +31,14 @@ class Options:
     """
     templates: Tuple[str, ...] = dataclasses.field(default=('DEFAULT',))
     namespace: Tuple[str, ...] = dataclasses.field(default=())
+    sample_configs: Tuple[str, ...] = dataclasses.field(default=())
     name: str = ''
+    sample_out_dir: str = 'samples'
+
+    # Class constants
+    SAMPLES_OPT: str = 'samples'
```

Comment:
I think @lukesneeringer mentioned we should use `gapic_samples`?

Reply:
I remember him retracting `gapic_samples` and preferring `samples`; let's wait for him to comment.